src/server/mod.rs

```diff
@@ -12,7 +12,7 @@ use ::api::SensorTemplate;
 use ::datastore;
 use ::sensors;
 
-mod handlers;
+pub mod handlers;
```

Comment:
We should move this to a `modifiers` module that is public, without publishing the handlers which aren't relevant for the end user.